CONTRIBUTING.md

```diff
@@ -7,30 +7,46 @@ If you just want to contribute a client library in a new language and not make c
 ## Requirements
 
 - MacOS or Linux (Windows may work too, but we haven't tested it)
-- NodeJS 12.x or 14.x
-- Yarn
-- Python 3.6+ (for the stats engine)
-  - scipy
-  - numpy
-  - pandas
-- Docker (for running MongoDB locally)
+- [NodeJS](https://nodejs.org/en/download/package-manager/) 12.x or 14.x
+  - Check version by running `node -v` on terminal
+- [Yarn](https://classic.yarnpkg.com/en/docs/install#windows-stable) 
+- [Python](https://www.python.org/downloads/) 3.6+ (for the stats engine)
+  - [scipy](https://scipy.org/install/)
+  - [numpy](https://numpy.org/install/)
+  - [pandas](https://pandas.pydata.org/docs/getting_started/install.html)
+- [Docker](https://docs.docker.com/engine/install/) (for running MongoDB locally)
+
+## Windows users
+
+One sure shot way to run GrowthBook on Windows is through installing [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/setup/environment#set-up-your-linux-user-info). These are some of the steps to follow, also outlined in the link above:
+  1. Search for your terminal app in the windows search bar
+  2. Select the option to "Run as administrator"
+  3. Now, on the terminal, run `wsl --install`
+  4. After the installation is complete, restart your computer
+  5. Set up your Linux username and password
+  6. Run `sudo apt update && sudo apt upgrade` (for Ubuntu or Desbian) to update and upgrade packages
+
+Now you have the basic Linux system set up, and can follow along with all the other steps
 
 ## Getting started
 
 1. Fork the project
 2. Clone your forked project by running `git clone git@github.com:{ YOUR_USERNAME }/growthbook.git`
-3. Run `cd growthbook`
-4. Run `yarn` to install dependencies
-5. Install [poetry](https://python-poetry.org/docs/master/#installing-with-the-official-installer)
-6. Run `yarn setup` to do the initial build
-7. Start MongoDB in Docker
+    - Can also use `git clone` and list the HTTPS URL of the repo afterwards  
+4. Run `cd growthbook`
+5. Run `yarn` to install dependencies
+6. Install [poetry](https://python-poetry.org/docs/master/#installing-with-the-official-installer)
+7. Run `yarn setup` to do the initial build
+8. Start MongoDB in Docker
```

Comment:
Looking at this now, maybe it's simpler to give two options: 

1. Install Mongo directly (no Docker) -  https://www.mongodb.com/docs/manual/installation/
2. Install Mongo with Docker - (code snippet below plus the help links)

If someone doesn't already have Docker installed and set up, installing Mongo directly is way easier.